setup.py

```diff
@@ -102,6 +102,7 @@ def run(self, *args, **kwargs):
         'install': CustomInstall
     },
     install_requires=install_reqs,
+    python_requires='>= 3.6',
```

Comment:
The way this is typically written is `>=3.6`, without the space. Could you change that, then I will merge.

Thanks!

Reply:
Space removed.